lib/sensu/settings/loader.rb

```diff
@@ -124,7 +124,7 @@ def load_file(file)
       def load_directory(directory)
         warning("loading config files from directory", :directory => directory)
         path = directory.gsub(/\\(?=\S)/, "/")
-        Dir.glob(File.join(path, "**/*.json")).each do |file|
+        Dir.glob(File.join(path, "**{,/*/**}/*.json")).uniq.each do |file|
```

Comment:
Seems we need a `uniq()` anyways, as this pattern along w/ alternatives, will pick up a symlink to a dir that it has already listed :+1: